packages/fluentui/react-northstar/src/themes/teams/components/Loader/loaderVariables.ts

```diff
@@ -33,7 +33,7 @@ export const loaderVariables = (): LoaderVariables => ({
     largest: pxToRem(72),
   },
 
-  svgContent: loaderSvgDataUrl,
+  svgContent: loaderSvgDataUrl(siteVariables.colorScheme.brand.border2, siteVariables.colorScheme.brand.foreground),
```

Comment:
it would be good to allow customization of this color - either using `variables` from v0 or using a css variable. @layershifter which one would you say is better?

Reply:
@jurokapsiar the problem is that we **can't** use CSS variables in `data:` urls. It was one of options that we tried before, but it does not work 😭 

My expectation on this refactor was that we will do a BC and will inline SVG into a component (like in v9):

![image](https://user-images.githubusercontent.com/14183168/195806117-c6e903e2-323c-44ff-8b79-e21199fa9e27.png)

➡️ Then we can use CSS variables for styling or apply styles to the SVG itself.